core/driver/sources/src/main/scala/org/virtuslab/ideprobe/Shell.scala

```diff
@@ -22,11 +22,11 @@ case class CommandResult(outSafe: String, err: String, exitCode: Int) {
   def isFailed: Boolean = !isSuccess
 
   def out: String = {
-    ok()
+    ensureIsOk()
     outSafe
   }
 
-  def ok(): Unit = {
```

Comment:
Oh! okay, I thought this class (`CommandResult`) is coming from an external library, not our codebase... hence I suggested an implicit class 😅 ofc in this case, a rename is enough

Reply:
Huh I don't think this can be put more concisely TBH

Reply:
Right. If so, maybe name it `assertSuccess()` for consistency with both BuildResult classes that we have. It will also be consistent with `isSuccess` method in this class.